docs/asset-converter.md

```diff
@@ -0,0 +1,98 @@
+# Asset converter
+
+Asset converter purpose is to convert assets from one format to another. Out of the box it supports conversion of
+several popular formats into JavaScript and CSS.
+
+## Configuring asset conversion
+
+In order to use asset conversion we have to configure it first. Let's see how it's done. As example
+let's convert [SCSS](https://sass-lang.com/) into CSS.  
+
+
+We'll use [foxy](https://github.com/fxpio/foxy). Since it calls npm we'll need [NodeJS](https://nodejs.org/en/) installed.
+After it is done, create `package.json`:
+
+```json
+{
+    "license": "BSD-3-Clause",
+    "dependencies": {
+        "sass": "^1.24.0"
+    }
+}
+```
+
+`npm install` brings three packages into `node_modules` directory of our application.
+
+Below we're using bootstrap bundle from "[Asset bundles](asset-bundles.md)" guide:
+
+```php
+namespace App\Assets;
+
+use Yiisoft\Assets\AssetBundle;
+
+/**
+ * Asset bundle for the Twitter bootstrap css files.
+ *
+ * BootstrapAsset.
+ */
+class BootstrapAsset extends AssetBundle
+{
+    public ?string $basePath = '@basePath';
+
+    public ?string $baseUrl = '@web/assets';
+
+    public ?string $sourcePath = '@npm/bootstrap';
+
+    public array $css = [
+        'scss/bootstrap.scss',
+    ];
+
+    public array $converterOptions = [
+        'scss' => '--style = compressed',
```

Comment:
I wonder if there's any reason for setting options to specific files rather than to all files of a type at once...

Reply:
I think it would be good to allow options of the converter by asset bundle now the classes are final and should be more flexible we can also create the global options in the converter, one more option will have no problem.

Reply:
There are global options in the converter already: https://github.com/yiisoft/assets/blob/master/src/AssetConverter.php#L35. You can easily set them with https://github.com/yiisoft/assets/blob/master/src/AssetConverter.php#L123

Reply:
Converter has all the options needed exposed. I don't think the fact that it's final harms the ability to configure it.

Reply:
> I think it would be good to allow options of the converter by asset bundle

What are exact use cases for it? Why would you set options per asset bundle?

Reply:
You may want to minify only one bundle, generate the css maps without going directly to the converter.

Reply:
OK.